test/libyul/objectCompiler/eof/rjumps.yul

```diff
@@ -0,0 +1,40 @@
+object "a" {
+  code {
+    if true {
+        mstore(0, 1)
+    }
+
+    return(0, 32)
+  }
+}
+
+// ====
+// EVMVersion: >=prague
+// bytecodeFormat: >=EOFv1
+// ----
+// Assembly:
+//     /* "source":49:53   */
+//   0x01
+//     /* "source":46:70   */
+//   rjumpi{tag_1}
+//     /* "source":22:112   */
+// tag_2:
+//     /* "source":93:95   */
+//   0x20
+//     /* "source":90:91   */
+//   0x00
+//     /* "source":83:96   */
+//   return
+//     /* "source":54:70   */
+// tag_1:
+//     /* "source":66:67   */
+//   0x01
+//     /* "source":63:64   */
+//   0x00
+//     /* "source":56:68   */
+//   mstore
+//     /* "source":54:70   */
+//   rjump{tag_2}
+// Bytecode: ef00010100040200010010040000000080ffff6001e1000460205ff360015f52e0fff5
+// Opcodes: 0xEF STOP ADD ADD STOP DIV MUL STOP ADD STOP LT DIV STOP STOP STOP STOP DUP1 SELFDESTRUCT SELFDESTRUCT PUSH1 0x1 RJUMPI 0x4 PUSH1 0x20 PUSH0 RETURN PUSH1 0x1 PUSH0 MSTORE RJUMP 0xFFF5
```

Comment:
Would be nice to eventually have a test type that makes verifying this on a bytecode level less tedious. For now I checked it by hand and it seems correct:
- `RJUMP 0xFFF5` jumps 3 bytes ahead and 11 back, which gives `13 + 3 - 11 = 5 = tag_2`
- `RJUMPI 0x4` jumps to `2 + 3 + 4 = 9 = tag_1`

```
00: PUSH1 0x1
02: RJUMPI 0x4
05: PUSH1 0x20 // tag_2
07: PUSH0
08: RETURN
09: PUSH1 0x1  // tag_1
11: PUSH0
12: MSTORE
13: RJUMP 0xFFF5
```

Reply:
I understand it as a comment for the future or we can do something with this already?